src/__private_stories__/text-preset-vars-story.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import {Stack, Text} from '..';
+import {vars} from '../skins/skin-contract.css';
+
+export default {
+    title: 'Private/TextPresets CSS vars',
+};
+
+const TextWrapper = ({
+    weight,
+    size,
+    lineHeight,
+    children,
+}: {
+    weight?: string;
+    size?: string;
+    lineHeight?: string;
+    children: React.ReactNode;
+}) => {
+    return (
+        <Text>
+            <span style={{fontWeight: weight, fontSize: size, lineHeight}}>{children}</span>
```

Comment:
why don't you pass these styles to `Text` using props?

Reply:
Discussed offline. We would need a "reverse mapping" from number to weight strings (bold/light/regular/medium). Given that this is a private story, I don't see any benefit. If there is a need in the future, we can reiterate on this